src/services/goals.js

```diff
@@ -1281,7 +1281,11 @@ async function createObjectivesForGoal(goal, objectives, report) {
   }
 
   // we don't want to create objectives with blank titles
-  return Promise.all(objectives.filter((o) => o.title).map(async (objective) => {
+  return Promise.all(objectives.filter((o) => o.title
+    || o.ttaProvided
+    || o.topics.length
+    || o.resources
+    || o.files).map(async (objective) => {
```

Comment:
why are these not checked in the same way as topics?

Reply:
good catch, thanks